### ISSUE TITLE
Slider: het eerste schermpje bevat een kop zonder content

### ISSUE THREAD

**Issue by julezrulez** [minvws/nl-covid19-notification-app-website#253]
Er is sinds deze week een afbeelding in de slider toegevoegd met een tekst in een H3. Deze kop kondigt content aan, echter die is er niet, want in de volgende slide staat alweer de volgende H3 klaar. (WCAG SC 1.3.1)

**Comment by anneke** [comment]
Hoi Jules! Bedankt voor je opmerking. Ik heb gelijk even gekeken, maar helaas heb ik zelf op dit vlak een gat in mijn kennis. Ik ga hier hulp bij zoeken! (mocht iemand deze issue zien en kunnen oppakken, heel erg welkom natuurlijk!)

**Comment by anneke** [comment]
Ik heb iets aangepast en dit probleem zou moeten zijn verholpen! Spoedig gaan we ook over naar een aangepaste homepage waar de slider verdwijnt. Ik sluit deze issue nu, maar zal je via de Fronteers slack nog even om feedback vragen!

**Comment by julezrulez** [comment]
Semantisch zou ik dit als volgt oplossen:
De eerste slideritem bevat een logo en een kop "Coronamelder".
- Geef de afbeelding een alt="". Hierdoor los je een deel van het kop-content probleem op (content boven de kop) en wordt het woord "Coronamelder niet 2x voorgelezen".
- Optie: houdt de H3 in de eerste slide intact en maak van elke volgende slide een H4. De H3 is welliswaar noet bijzonder beschrijvend, maar in het geheel klop de boel dan semantisch.

H2 Dit is wat je kunt verwachten van de app
H3 CoronaMelder
H4 Bescherm je vrienden, familie en de rest van Nederland
H4 Je krijgt een melding nadat je extra kans op besmetting hebt gelopen.
H4 CoronaMelder gebruikt Bluetooth, geen locatiegegevens
H4 Voorbeeld Fietste iemand voorbij? Dan krijg je later geen melding
H4 Voorbeeld Zat je dicht bij iemand in de
